ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/analytics/model/Constants.kt

```diff
@@ -1,3 +1,6 @@
 package org.cru.godtools.tutorial.analytics.model
 
 internal const val ADOBE_SITE_SECTION_TUTORIAL = "tutorial"
+internal const val ADOBE_SITE_SECTION_ON_BOARDING = "on_boarding"
+internal const val BASE_SCREEN_NAME_ON_BOARDING = "onboarding"
+internal const val BASE_SCREEN_NAME_TUTOIRAL = "tutorial"
```

Comment:
the `BASE_SCREEN_NAME_*` constants are unncessary

Reply:
@frett so are you saying the constants should only be named `ON_BOARDING` and  `TUTORIAL`.

Reply:
I'm saying it was unnecessary to make these constants.

Reply:
the existence of these constants I don't feel strongly one way or another, but in the context of the current usage it was unnecessary to move these values to constants

Reply:
we really should try to exclude unnecessary changes from PRs, it will reduce the amount of code that someone needs to look at when reviewing the PR

Reply:
@frett Removing the constants will mean that I will have to evaluate the `when` on a string (example below).  Not sure if that is the best solution.

```
 override val adobeSiteSection
        get() = when (tutorial.analyticsBaseScreenName) {
            "onboarding" -> ADOBE_SITE_SECTION_ON_BOARDING
            "tutorial" -> ADOBE_SITE_SECTION_TUTORIAL
            else -> null
        }
```

Reply:
Not if you change the when to switching on the enums like the other comment suggested

Reply:
@frett Are you saying just set the `adobeSiteSection = tutorial.analyticsBaseScreenName`

Reply:
No, I'm saying you shouldn't need to use `analyticsBaseScreenName` at all for the adobe site section